packages/sdks/output/vue/vite.config.js

```diff
@@ -19,17 +19,22 @@ export default defineConfig({
         const outPath = getSdkOutputPath();
         copyFileSync(outPath + '/index.d.ts', outPath + '/index.d.mts');
         renameSync(outPath + '/index.d.ts', outPath + '/index.d.cts');
+        copyFileSync(outPath + '/functions/evaluate/node-runtime/init.d.ts', outPath + '/init.d.mts');
+        copyFileSync(outPath + '/functions/evaluate/node-runtime/init.d.ts', outPath + '/init.d.cts');
       },
     }),
   ],
   build: {
     lib: {
-      entry: resolve(__dirname, 'src/index.ts'),
+      entry: {
+        index: resolve(__dirname, 'src/index.ts'),
+        init: resolve(__dirname, 'src/functions/evaluate/node-runtime/init.ts')
```

Comment:
the issue with this approach is that it also creates `lib/edge/init.mjs` and `lib/browser/init.mjs`, both of which will import `isolated-vm`. This can cause issues and confusion.

We want to makes sure the ivm import is nowhere within those two bundles.


'![CleanShot 2024-09-30 at 15 02 24@2x](https://github.com/user-attachments/assets/fedb1582-5d7e-4a4c-b485-841f8c1a1aa1)

Reply:
you can workaround this by checking the `SDK_ENV` and only adding this entry point for `node`

Reply:
thanks for the catch, done! we also need to do this for React SDK and Qwik SDK next